ccd-definition/AuthorisationCaseField/AuthorisationCaseField-JO-nonprod.json

```diff
@@ -168,5 +168,15 @@
       "caseworker-civil-staff"
     ],
     "CRUD": "CRU"
-  }
+  },
+  {
+  "CaseTypeID": "CIVIL",
+  "CaseFieldID": "joIssuedDate",
+  "UserRoles": [
+    "caseworker-civil-admin",
+    "caseworker-civil",
```

Comment:
"caseworker-civil" permission should be reviewed. if it should be reivewed then update it for whole event "add judgement"